mqtt_subscriber/device_profile_player/main.py

```diff
@@ -0,0 +1,65 @@
+import os
+import logging
+import json
+import asyncio
+from time import time
+import websockets
+from pendulum import now, duration
+from d3a_interface.utils import from_timestamp
+from d3a_interface.read_user_profile import read_profile_without_config
+
+
+class DeviceProfilePlayback:
+
+    def __init__(self):
+        profile_path = os.environ['DEVICE_PLAYBACK_PROFILES_PATH']
+        self._year_to_replay = int(os.environ['DEVICE_PLAYBACK_YEAR_TO_REPLAY'])
+        csv_files = self._list_all_available_filenames_and_paths(profile_path)
+        self.profile_dicts = {
+            filename.split(".")[0]: read_profile_without_config(filepath)
+            for filename, filepath in csv_files
+        }
+
+    @staticmethod
+    def _list_all_available_filenames_and_paths(profile_path):
+        filenames = os.listdir(profile_path)
+        return [(filename, os.path.join(profile_path, filename))
+                for filename in filenames
+                if filename.endswith(".csv")]
+
+    async def _scheduled_call(self, websocket):
+        current_timestamp = now().timestamp()
+        offset_seconds = current_timestamp % 900
```

Comment:
```suggestion
        offset_seconds = current_timestamp % RELOAD_CN_DEVICE_LIST_TIMEOUT_SECONDS
```

May be a better name for constant variable, just pasted it here as its also `900sec` equivalent to standard one market interval.

Reply:
Have a look again, I just committed a change that makes it configurable.